docs/getting_started/installation.md

```diff
@@ -1,48 +1,7 @@
 # Installation
 
-## Portable Standalone
+To install Shift, download the *.zip* file corresponding to the preferred Python version from the [website](https://https://www.inbibo.co.uk/shift). Shift distributions packages come with a built-in Python interpreter, along with all necessary third-party libraries to use Shift and its native catalogs.
```

Comment:
@mromeoinbibo will we also have `zip` for the linux version or will it be a different format?

Reply:
Should I add this question to card #44? At the moment we are not adding the section for Linux

Reply:
Yes, please